_posts/2021-05-04-plume-oc.markdown

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Plume, an original character"
+date: 2021-05-04 22:00
+categories: art oc painting
+---
+
+{::options parse_block_html="true" /}
+
+![A drawing of a person with hair in nonbinary flag colours, hugging a pillow.](/assets/plume-oc/2021-05-03-plume-icon-no-bg.png)
+
+<span style="font-size:80%;">"Plume" © 2021 Komi Amiko, published under [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/) | [version with background](/assets/plume-oc/2021-05-03-plume-icon-bg.png) | [download assets](/assets/plume-oc/2021-05-03-plume-icon-source.tar.bz2)</span>
+
+# Serious
+{: #serious}
+
+{% include collapser.markdown %}
+
+<div>
+
+Before I begin, I would like to say that, in my opinion, the English rule of putting punctuation inside quotes is stupid and it should not exist.
+Punctuation carries meaning, and if you want to quote some text exactly, you should not arbitrarily change the punctuation inside the quote because some rule said to for aesthetic reasons.
+
+I have always been "bad at art".
+I have tried a variety of forms of art, including 3D animation, digital painting, and music composition, and I have only been mediocre at all of them.
```

Comment:
This is entirely for aesthetic/how it sounds purposes but instead of `forms of art` you could perhaps say `art forms`.